### PR TITLE
Make smooth on hover

### PR DESCRIPTION
**Pull request by sv-dubov**
- [DEMO LINK](https://sv-dubov.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
           </li>
 
           <li class="nav__item">
-            <a href="#sampsung" class="nav__link">Samsung</a>
+            <a href="#samsung" class="nav__link">Samsung</a>
```
Comment (nzrii-kysel) [inline src/index.html:31]: love this

File: src/styles/utils/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$transition-duration: 300ms;
+$transform-scale: scale(1.2);
```
Comment (nzrii-kysel) [inline src/styles/utils/_variables.scss:2]: do you really need a variable that is only used once?

**Review by nzrii-kysel — APPROVED** [review]
well done

File: src/styles/utils/_variables.scss (hunk 2 above)
Comment (ArturErmolenko) [inline src/styles/utils/_variables.scss:2]:
```suggestion
```

File: src/index.html (hunk 1 above)
Comment (ArturErmolenko) [reply src/index.html:31]: need to fix it

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (sv-dubov) [reply src/index.html:31]: @ArturErmolenko Do I need to return "sampsung"?

File: src/styles/utils/_variables.scss (hunk 2 above)
Comment (sv-dubov) [reply src/styles/utils/_variables.scss:2]: I thought I would use it anywhere more. But I didn't :)

File: src/index.html (hunk 1 above)
Comment (alexandra-protyanova) [reply src/index.html:31]: nope 😉

**Review by alexandra-protyanova — APPROVED** [review]
Good job!

**Pull request closed without merging**